eth/testdata/service_mock.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"github.com/ethereum/go-ethereum/eth"
+)
+
+type Service struct {
+
+}
+
+func (s *Service) Namespace() string {
+	return "test"
+}
+
+func (s *Service) Version() string {
+	return "1.0.0"
+}
+
+type API struct {
+	eth *eth.Ethereum
```

Comment:
`Ethereum` is not a stable API right? Also seems like a plugin could really mess things up with unfettered access to the node like this.

Reply:
Yeah I am currently question what the best thing is to pass to ensure plugins can be given all the data they need to be useful.